projects/go-lib/src/lib/components/go-select/go-select.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input, ViewEncapsulation } from '@angular/core';
+import { FormControl } from '@angular/forms';
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Do we _have_ to do this? If so, what's the reasoning for it?

Reply:
Sadly, yes. Otherwise we would have to import the ng select stylesheet to our styles.scss as well. This was a way around. I'm open for other suggestions